governance-with-multisig/ganache.sh

```diff
@@ -0,0 +1,2 @@
+mkdir -p ganache
+ganache-cli -l 6000000 -e 1000 -i 8888 --db ganache -p 9545 -d
```

Comment:
I try to use long options in scripts, so that the intent is clear to unfamiliar users (e.g. `--deterministic` instead of -d`).